src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/PaymentMessageParser.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus;
+
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.microsoft.azure.servicebus.MessageBody;
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.exceptions.InvalidMessageException;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.model.PaymentMessage;
+
+import java.io.IOException;
+import java.util.List;
+
+public class PaymentMessageParser {
+
+    private static final ObjectMapper objectMapper;
+
+    static {
+        objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        objectMapper.enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
```

Comment:
Do we need to be case-insensitive with enums? It's APIs talking to each other - they should use exact values. I think it's best to be strict.

Reply:
it is same as orchestrator. how about  javatimemodule do we need to use  ?

Reply:
Why? Shouldn't we have it consistent all over the application?

Reply:
We only need java time module if we serialise/deserialise date(time)s. Not sure if we're doing it here.